spec/widgets/histogram/content-view.spec.js

```diff
@@ -12,7 +12,8 @@ describe('widgets/histogram/content-view', function () {
     });
 
     this.widgetModel = new HistogramWidgetModel({
-      title: 'Howdy'
+      title: 'Howdy',
+      attrsNames: ['title']
```

Comment:
Nice question. I think is something we need to change. 

`attrsNames` contains the attributes allowed to be changed. I think this is something good but should be defined as an internal property on each widgetModel instead to be setted on widgetModel createion. (See https://github.com/CartoDB/deep-insights.js/blob/master/src/widgets-service.js#L62)